src/app/view/components/main-legacy/main-legacy.component.ts

```diff
@@ -18,6 +21,44 @@ export class MainComponentLegacy implements OnInit, OnDestroy {
     ngOnInit(): void {
         document.body.style.backgroundColor = '';
         document.body.style.backgroundImage = '';
+
+        this.isTabletScreenOrLess();
+    }
+
+    /**
+     * Set collapsed menu if window size is less than iPad landscape size
+     * @memberof MainComponentLegacy
+     */
+    isTabletScreenOrLess(): void {
+        if (window.innerWidth < 1025) {
+            this.isTablet = true;
+            this.isMenuCollapsed = true;
+        }
+    }
+
+    /**
+     * Set isTablet when resizing the window size
+     * @param {any} event
+     * @memberof MainComponentLegacy
+     */
+    @HostListener('window:resize', ['$event'])
+        onResize(event: any) {
+        this.isTablet = event.target.innerWidth < 1025;
+    }
+
+    /**
+     * Respond to document events and collapse the sidenav if is clicked outside
+     * @param {*} event
+     * @memberof DotNavigationComponent
+     */
+    @HostListener('document:click', ['$event'])
+    onClickOutside(event: any) {
+        const eTarget = event.target;
+
+        // tslint:disable-next-line:max-line-length
+        if (!this.mainNav.sideNavEl.nativeElement.contains(eTarget) && eTarget.className !== 'layout__sidebar' && eTarget.className.split(' ')[1] !== 'ui-clickable') {
```

Comment:
this 'this.mainNav.sideNavEl.nativeElement.contains(eTarget)' should be a mainNav method, something like: this.sideNavElContains(eTarget)